eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/Binary/Annotator.hs

```diff
@@ -127,37 +122,23 @@ mapTraverseableDecoderA ::
   Decoder s (Annotator (m b))
 mapTraverseableDecoderA decList transformList = fmap transformList . sequence <$> decList
 
-deriving via
-  Mem (ShelleyTxRaw era)
-  instance
-    ( EraTx era
-    , DecCBOR (Annotator (TxBody era))
-    , DecCBOR (Annotator (TxWits era))
-    , DecCBOR (Annotator (TxAuxData era))
-    ) =>
-    DecCBOR (Annotator (ShelleyTx era))
-
 instance
   ( EraTx era
   , DecCBOR (Annotator (TxBody era))
   , DecCBOR (Annotator (TxWits era))
   , DecCBOR (Annotator (TxAuxData era))
   ) =>
-  DecCBOR (Annotator (ShelleyTxRaw era))
+  DecCBOR (Annotator (ShelleyTx era))
   where
   decCBOR =
     decode $
-      Ann (RecD ShelleyTxRaw)
+      Ann (RecD ShelleyTx)
         <*! From
         <*! From
-        <*! D
-          ( sequence . maybeToStrictMaybe
-              <$> decodeNullMaybe decCBOR
-          )
+        <*! Ann (D (decodeNullStrictMaybe decCBOR))
```

Comment:
After giving a bit more thought this is actually wrong, because then instead of `DecCBOR (Annotator (TxAuxData era))` it will use `DecCBOR (TxAuxData era)`. We would have caught this eventually during the move of `Annotator` instances out of `testlibs`, but it is better not to add this mistake in the first place
```suggestion
        <*! D (sequence <$> decodeNullStrictMaybe decCBOR)
```